bin/pygrb/pycbc_pygrb_plot_chisq_veto

```diff
@@ -76,8 +76,10 @@ def load_data(input_file, vetoes, ifos, opts, injections=False):
         if snr_type in ['coherent', 'null', 'reweighted']:
             data[snr_type] = trigs_or_injs['network/%s_snr' % snr_type][:]
         elif snr_type == 'single':
-            att = opts.ifo[0].lower()
-            data[snr_type] = trigs_or_injs['%s/snr_%s' % (opts.ifo, att)][:]
+            key = opts.ifo + '/snr_' + opts.ifo.lower()
+            if opts.ifo.lower() != 'h1':
```

Comment:
Why is H1 being hardcoded here? Is this temporary? Add a comment please.

Reply:
It is undoing this: https://github.com/gwastro/pycbc/blob/64ecd78aeea6cf2605a5198684c30782c93448c0/pycbc/events/eventmgr.py#L710.  I will add a comment.

Reply:
Change eventmgr.py if you need to! The Class you point to was written for the GRB analysis, and is only used by GRB. It looks like it's following an old XML standard (as changing XML column names was always a disaster), and we don't need to follow those rules in O4.

.... I think the idea was that the SNR would be stored in columns `snr_h1`, `snr_h2`, `snr_l`, `snr_v`, and hence H1 was treated specially (and we hope no-one ever tries to run this on old H2 data, I guess!!) I would prefer using the full ifo name (`snr_l1`, `snr_v1`, to avoid having to do conversions all the time).